client/src/components/UserProfile.tsx

```diff
@@ -46,16 +44,8 @@ const Profile = (): React.ReactElement => {
           <Avatar
             sx={{ width: 80, height: 80 }}
             src={profileData!.picture !== null ? profileData!.picture : ''}
-            alt={
-              profileData!.username !== null
-                ? profileData!.username
-                : profileData!.name !== null
-                ? profileData!.name
-                : ''
-            }
-          >
-            {/*profileData!.username![0]*/}
-          </Avatar>
+            alt={profileData!.username ? profileData!.name : ''}

```

Comment:
This checks if the **username** exists and sets the alt to the **name** or an empty string. This doesn't make much sense to me.